parm/land_analysis.yaml

```diff
@@ -8,7 +8,9 @@ workflow:
       spec: 201912210000 201912210000 24:00:00
```

Comment:
I'm curious why there are both yaml and xml files in the repository. The yaml file should be able to create the XML using `uwtools` so that the static XML does not need to be version controlled.

Reply:
@christinaholtNOAA. good point. You're correct. That xml file should be a temporary one. Currently, `uwtool` has not been integrated into the repo yet. To be honest, I am not familiar with `uwtool` yet. :) Once `uwtool` is fully integrated, we'll remove this xml file from the repo.

Reply:
we can clean up xml later.